sdp.c

```diff
@@ -22,69 +24,39 @@
 #include "debug.h"
 
 
-static su_home_t *home = NULL;
-
-
 /* SDP Initialization */
 int janus_sdp_init(void) {
-	home = su_home_new(sizeof(su_home_t));
```

Comment:
You mean the `janus_sdp_init()` and `janus_sdp_deinit()`? I kept them as placeholders, in case something comes up that needs setup there. But yeah, they're pretty much useless right now.